share/wake/lib/system/remote_cache_api.wake

```diff
@@ -336,7 +332,12 @@ export def rscApiPostStringBlob (name: String) (value: String) (api: RemoteCache
 export def rscApiPostFileBlob (name: String) (file: String) (api: RemoteCacheApi): Result String Error =
     # We must use unsafe here since we cannot elevate *file* to a Path without either copying it
     # or triggering the 'job output by multiple files' error.
-    uploadBlobRequest api (unsafe_addFormData name file None)
+
+    def contentType = match (unsafe_stat file)
+        Pass (Stat _ _ size) -> contentTypeFromSize size
+        Fail _ -> None
```

Comment:
Wondering if there is some benefit to failing earlier here rather than waiting for the `uploadBlobRequest` to fail if it can't stat the file?

Reply:
Maybe, I'm not sure which one happens first but I'll at it to the list of things to give a look